cmd/ko/README.md

```diff
@@ -181,6 +189,36 @@ to whatever `kubectl` context is active.
 `ko delete` simply passes through to `kubectl delete`. It is exposed purely out
 of convenience for cleaning up resources created through `ko apply`.
 
+
+### With `minikube`
```

Comment:
I'd pull this up to `##` since the `###` headings under `Usage` act as a list of commands.